Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Identifier
+    static let identifier = Identifiers.gymWeekCell
+
+    // MARK: - Views
+    private var dayLabel: UILabel!
+    private var backCircle: UIView!
+
+    // MARK: - Info
+    /// Weekday this cell represents
+    private var day: WeekDay?
+    /// Weekday representing current day of week
+    private var today: WeekDay?
```

Comment:
it seems like the only place we use `today` in this file is on line 83 where we use it to check if `day == today`. We should just pass in a `isToday: Bool` in the configure and store that instead of a `WeekDay` object